File: module_login/src/main/java/com/ashlikun/baseproject/module/debug/ModuleDebugApp.java
package com.ashlikun.baseproject.module.debug;

import com.ashlikun.baseproject.libcore.BaseApplication;
import com.ashlikun.baseproject.module.login.ModuleApp;
import com.ashlikun.baseproject.common.CommonApp;

/**
 * 作者　　: 李坤
 * 创建时间: 2018/12/14　15:50
 * 邮箱　　：496546144@qq.com
 * <p>
 * 功能介绍：
 */
public class ModuleDebugApp extends BaseApplication {
    @Override
    public void onCreate() {
        addApplication(new CommonApp());
        addApplication(new ModuleApp());
        super.onCreate();
    }
}

File: module_login/src/main/java/com/ashlikun/baseproject/module/debug/ModuleDebugApp.kt
package com.ashlikun.baseproject.module.debug

import com.ashlikun.baseproject.libcore.BaseApplication
import com.ashlikun.baseproject.module.login.ModuleApp
import com.ashlikun.common.CommonApp

/**
 * @author　　: 李坤
 * 创建时间: 2018/7/25 10:49
 * 邮箱　　：496546144@qq.com
 *
 * 功能介绍：用于单独运行登录模块的Application,正式打包的时候是没有这个类
 */

class ModuleDebugApp : BaseApplication() {

    override fun onCreate() {
        addApplication(CommonApp())
        addApplication(ModuleApp())
        super.onCreate()
    }
}
